QuestTracking.lua

```diff
@@ -6,7 +6,7 @@ local hadquest
 
 
 TourGuide.TrackEvents = {"UI_INFO_MESSAGE", "CHAT_MSG_LOOT", "CHAT_MSG_SYSTEM", "QUEST_LOG_UPDATE", "ZONE_CHANGED", "ZONE_CHANGED_INDOORS",
-	"ZONE_CHANGED_NEW_AREA"}
+	"ZONE_CHANGED", "ZONE_CHANGED_NEW_AREA"}
 
```

Comment:
Not sure if this is significant for the operation, but I think it served the similar purpose as MINIMAP_ZONE_CHANGED.

Reply:
Ah, did not even notice. True.

Reply:
Commit for this removed.